src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+for (let i = 0; i < table.rows.length; i++) {
+  const row = table.rows[i];
+  const cells = row.cells[1].innerHTML;
+  const newCells = row.insertCell(row.cells.length - 1);
+
+  if (
+    row.parentNode.tagName === 'THEAD' ||
+    row.parentNode.tagName === 'TFOOT'
+  ) {
+    newCells.outerHTML = `<th>${cells}</th>`;
+  } else {
+    newCells.outerHTML = `<td>${cells}</td>`;
```

Comment:
Code Quality Issue: The code in the `if` and `else` branches is almost identical, with the only difference being the tag name (`th` vs `td`). This is a sign that the code could be made more DRY (Don't Repeat Yourself). You could define the tag name in a variable depending on the condition, and then use this variable when setting `newCells.outerHTML`.